global.json

```diff
@@ -3,7 +3,12 @@
     "version": "3.0.100-preview5-011568"
   },
   "tools": {
-    "dotnet": "3.0.100-preview5-011568"
+    "dotnet": "3.0.100-preview5-011568",
+    "runtimes": {
+      "dotnet": [
+        "$(MicrosoftNETCoreAppPackageVersion)"
```

Comment:
global.json can now resolve version numbers from MSBuild files?

Reply:
It's a custom Arcade feature. It's not generally available to the rest of the world.